OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -250,60 +249,60 @@ public SubCell GetAvailableSubCell(World world, Actor self, CPos cell, SubCell p
 
 	public class Mobile : IIssueOrder, IResolveOrder, IOrderVoice, IPositionable, IMove, IFacing, ISync, INotifyAddedToWorld, INotifyRemovedFromWorld, INotifyBlockingMove
 	{
-		public readonly Actor self;
+		public readonly Actor Self;
```

Comment:
Does anything else actually reference this?  It would be better to keep this private and lower case.